package ExceptionHandling;

public class Testingtype {
public static void main(String[] args) {
	try {
		System.out.println("hello");
	}
	catch(ArithmeticException i) {
		
	}
}
}
